config/test_fixtures.py

```
@@ -0,0 +1,302 @@
+    yield loop
+    loop.close()
+
+
+class ServiceTestHelper:
+    """Helper class for testing service interactions."""
+
+    def __init__(self, test_settings):
+        self.settings = test_settings
+
+    def wait_for_service_ready(
+        self, service_name: str, health_check_url: str, timeout: int = None
```

Ruff (0.13.3)

Unused method argument: `health_check_url`

(ARG002)